deployments/engine/V3/internal-testing/staging-v3-react-demo-2/DEPLOYMENTS.md

```diff
@@ -0,0 +1,42 @@
+
+# Deployment
+
+Date: 2023-05-04T19:39:28.024Z
+
+## **Network:** goerli
+
+## **Environment:** staging
+
+**Deployment Input File:** `deployments/engine/V3/internal-testing/staging-v3-react-demo-2/deployment-config.staging.ts`
+
+**GenArt721CoreV3_Engine:** https://goerli.etherscan.io/address/0xFb2480C9c02F3222d1299330976DACEB98009800#code
+
+**AdminACLV1:** https://goerli.etherscan.io/address/0xeA28d75dd6e3108000E4d0E1411A58b872248024#code
+
+**Engine Registry:** https://goerli.etherscan.io/address/0xEa698596b6009A622C3eD00dD5a8b5d1CAE4fC36#code
+
+**MinterFilterV1:** https://goerli.etherscan.io/address/0x682A5aB4d364a5CB5Ee372F6F5727C91431B8F10#code
+
+**Minters:**
+
+**MinterSetPriceV4:** https://goerli.etherscan.io/address/0xE4aF88B3E9a3045537339Fc05fd5F211fBFD92Df#code
+
+
+
+**Metadata**
+
+- **Starting Project Id:** 0
+- **Token Name:** V3 Engine Demo External Reader Lib
+- **Token Ticker:** DEMO
+- **Auto Approve Artist Split Proposals:** true
+- **Render Provider Address, Primary Sales:** deployer
+- **Platform Provider Address, Primary Sales:** deployer
+
```

Comment:
I think it would be good to update the script to add a line like:
```
- **BytecodeStorageReader:** <bytecodeStorageLibraryAddress>
```

Reply:
Great call – added to the engine deployment script.